### PR TITLE
Maniaplanet + PyPlanet egg

### PR DESCRIPTION
**Pull request by treier57lol**
- Adds Maniaplanet + PyPlanet egg
- Fully functional but might have some things to tweak

### CODE REVIEW

**Comment by NoxInmortus** [comment]
Interresting. PyPlanet seems to be the same kind of thing as Maniacontrol ?

What's the size of the generated docker image with python/pyplanet ?

**Comment by NoxInmortus** [comment]
Also, I can't import pterodactyl/pyplanet/egg-trackmania.json, ptero send me a http error 500, what's your version of Pterodactyl ? Mine may be too old (1.0.3)

File: pterodactyl/pyplanet/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# ----------------------------------
+# Pterodactyl Core Dockerfile
+# Environment: glibc + libuuid
+# Minimum Panel Version: 0.6.0
+# ----------------------------------
+FROM frolvlad/alpine-glibc:alpine-3.12
+
+LABEL maintainer='NoxInmortus'
+
+RUN apk add --update --no-cache wget ca-certificates libstdc++ musl libuuid git bash build-base libffi-dev openssl-dev bzip2-dev zlib-dev readline-dev sqlite-dev xz-dev postgresql-dev libpq \
```
Comment (NoxInmortus) [inline pterodactyl/pyplanet/Dockerfile:10]: You seem to install a lot of packages but i'm not sure they are necessary.

The documentation does not mention all of thoses : https://pypla.net/en/latest/intro/installation-linux.html

eg postgresql-dev  is out of place here as pterodactyl offer only mysql support. I would prefer you double check the required packages, we only want a minimal install.

Comment (treier57lol) [reply pterodactyl/pyplanet/Dockerfile:10]: Yes, I didn't have time to check what is needed, most of those are used during the install procedure and are not needed in the Docker image itself when running the server.

Comment (treier57lol) [reply pterodactyl/pyplanet/Dockerfile:10]: Yes, postgresql-dev is not needed, the install had an error with not finding pg_config and I added it to add that but it actually gets included in libpq, which is needed.

**Comment by treier57lol** [comment]
> Also, I can't import pterodactyl/pyplanet/egg-trackmania.json, ptero send me a http error 500, what's your version of Pterodactyl ? Mine may be too old (1.0.3)

Indeed, you need the latest version, v1.2.0, as they changed how Docker images are listed in the file and an old version can't read it so it gives an error 500.

**Comment by treier57lol** [comment]
I will not merge these changes as they include changes you completely rejected in my other PR.

**Pull request closed without merging**